network.md

```diff
@@ -226,6 +226,14 @@ Backpressure mechanism
 - syn-ack (receiver to sender) // ack the segment number received
 - ack (sender to receiver) // ack the segment number received
 
+## TCP Connection Termination
+
+4-way handshake
+- FIN (active close to passive close)
+- ACK (passive close to active close)
+- FIN (passive close to active close)
+- ACK (active close to passive close)
```

Comment:
Can we reuse how it's done for the TCP handshake? So that it keeps a certain level of consistency for the cards.

Reply:
OK

Reply:
My point was not to remove this card, it was to write it in the same way as the TCP handshake card has been written so make the cards consistent.

Reply:
Sorry for misunderstanding your previous review comment, I will fix it.